CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Modified
+
+- Migrated to chart dependency
+- ⚠ Major upgrade, breaking changes
+  - values structure changes. We rely on a subchart, meaning most of previous setup goes to a `promtail` section
```

Comment:
"most of" or "all of"?

I guess "most of" here comes from the Loki example. I had an extra line with "except..." to list what should not be moved to a subsection.

As a user, when I read "most of" without any more explanation I feel I lack some information.

Reply:
Changed "most of" to "all of"